test/asyncHook.test.ts

```diff
@@ -82,42 +82,66 @@ describe("async hook tests", () => {
   });
 
   test("should wait for expectation to pass", async () => {
-    const { result, wait } = renderHook(() =>
+    const { result, waitFor } = renderHook(() =>
       useSequence("first", "second", "third")
     );
 
     expect(result.current).toBe("first");
 
     let complete = false;
-    await wait(() => {
+    await waitFor(() => {
       expect(result.current).toBe("third");
       complete = true;
     });
     expect(complete).toBe(true);
   });
 
+  test.only("should wait for arbitrary expectation to pass", async () => {
```

Comment:
The `.only` here should have been removed 😓 

```suggestion
  test("should wait for arbitrary expectation to pass", async () => {
```

Removed in #12